src/authenticationservices.cs

```diff
@@ -651,6 +668,10 @@ interface ASAuthorizationSingleSignOnRequest {
 
 		[Export ("authorizationOptions", ArgumentSemantic.Copy)]
 		NSUrlQueryItem[] AuthorizationOptions { get; set; }
+
+		[NoWatch, NoTV, Mac (12,0), iOS (15,0), MacCatalyst (15,0)]
```

Comment:
minor: there's already a `[NoWatch, NoTV]` on the type so it's not needed on the members